src/BenchmarkDotNet/Toolchains/NativeAot/Generator.cs

```diff
@@ -150,7 +151,7 @@ protected override void GenerateProject(BuildPartition buildPartition, Artifacts
     <ProjectReference Include=""{GetProjectFilePath(buildPartition.RepresentativeBenchmarkCase.Descriptor.Type, logger).FullName}"" />
   </ItemGroup>
   <ItemGroup>
-    <RdXmlFile Include=""rd.xml"" />
+    {GetRdXmlFiles(buildPartition.RepresentativeBenchmarkCase.Descriptor.Type, logger).Select(_ => $"<RdXmlFile Include=\"{_}\" />{Environment.NewLine}")}
```

Comment:
nit `_` is typically used when the argument is ignored. In this case, we are using it.

```suggestion
    {GetRdXmlFiles(buildPartition.RepresentativeBenchmarkCase.Descriptor.Type, logger).Select(file => $"<RdXmlFile Include=\"{file}\" />{Environment.NewLine}")}
```